### PR TITLE
Page structure user

### PR DESCRIPTION
**Pull request by Yurii-Vipshovsky**
Fix footer view, because it was displayed over main page

### CODE REVIEW

File: sports_hub/sports_hub/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -3,46 +3,139 @@
...
-                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse" aria-controls="navbarSupportedContent"
-                        aria-expanded="false" aria-label="Toggle navigation">
-                    <span class="navbar-toggler-icon"></span>
+        <nav class="header fixed-top">
+            <button type="button" class="btn home" asp-area="" asp-controller="Home" asp-action="Index">
+                Sports Hub
+            </button>
+            <form class="form-inline header">
+                <button class="btn my-2 my-sm-0" type="submit" style="border:0px;">
+                    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ZarechOleksii) [inline sports_hub/sports_hub/Views/Shared/_Layout.cshtml:22]: SVG в окремий файл

File: sports_hub/sports_hub/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -2,7 +2,348 @@
...
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">VIDEO</a>
+            </li>
+        </ul>
+
+        <div class="follow" style="margin-top:70px; font-size:16px; color:#B2B2B2; padding:16px;">
+            Follow
+            <div class="follow ico" style="display:grid; grid-template-columns:40px 40px; grid-template-rows:40px 40px; margin-top:20px;">
+                <svg width="9" height="17" viewBox="0 0 9 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ZarechOleksii) [inline sports_hub/sports_hub/Views/Home/Index.cshtml:54]: SVG в окремі файли

File: sports_hub/sports_hub/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -2,7 +2,348 @@
...
+                </svg>
+
+            </div>
+        </div>
+    </div>
+
+
```
Comment (ZarechOleksii) [inline sports_hub/sports_hub/Views/Home/Index.cshtml:74]: Порожні рядки

**Review by ZarechOleksii — APPROVED** [review]

File: sports_hub/sports_hub/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -3,46 +3,139 @@
...
+                </svg>
```
Comment (XashDM) [inline sports_hub/sports_hub/Views/Shared/_Layout.cshtml:42]: Separate file for svg

File: sports_hub/sports_hub/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -3,46 +3,139 @@
...
+    <div style="margin-left:10px; margin-right:100px; margin-top:160px;">
         <main role="main" class="pb-3">
             @RenderBody()
         </main>
     </div>
 
-    <footer class="border-top footer text-muted">
-        <div class="container">
-            &copy; 2021 - sports_hub - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+    <footer class="footer">
```
Comment (XashDM) [inline sports_hub/sports_hub/Views/Shared/_Layout.cshtml:90]: Serhii also was doing footer please communicate with him if something needs to be adjusted

**Pull request merged**